snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs

```diff
@@ -59,7 +59,7 @@ static byte[] encryptStringToBytes_AesManaged(string plainText, byte[] Key, byte
                 AesManagedAlg.Key = Key;
                 AesManagedAlg.IV = IV;
 
-                // Create an encrytor to perform the stream transform.
+                // Create anencryptor to perform the stream transform.
```

Comment:
@mairaw sorry I'm here again, but there should be a white space inside *anencryptor*
The same is true for other three updates.

Reply:
OMG! What’s wrong with me? 😂

Reply:
Maybe, Sunday :)